### DISCUSSION TITLE
Head-related transfer function (HRTF) customization

### DISCUSSION THREAD

**Discussion by mfbolus** [losnoco/Cog#409] [Q&A]
First of all, thank you for your hard work on this project!

This is a "newb" question, but how do we direct the `Cog` app to look at a different (e.g. user's custom) HRTF? I see the option to toggle the filter on/off, but where would the user put a personalized HRTF?

**Comment by kode54** [comment]
Do you have a full sphere HRTF set in OpenAL format, preferably sampled at 96kHz? I can add support for loading your own preset file, similar to what I did with the icon set support. A checkbox to turn it on if present, and a button to install a preset to the configuration directory.

The bundled preset is in Cog.app/Contents/Resources, as `SADIE_D02-96000.mhr`

It should also include angles above and/or below level with the listener.

**Comment by mfbolus** [comment]
I have not yet made an HRTF but am working on it. It's very helpful to know the specifications needed, so thank you! Frankly, I am trying to see if there is any merit to spatial audio mixed down to binaural (vs. stereo). So, this is in the experimental phase and I honestly would not expect you to add functionality unless others would use it.  I am just thankful Cog supports spatial audio playback. While I am trying things, I will just replace the HRTF file it expects under Resources.
